requirements.txt

```diff
@@ -1,4 +1,4 @@
-# python_version 3.7.6
```

Comment:
in other libs
`# pip_version 20.*`
was added. Why not here?

Reply:
why the python version has been updated here, but not in other libs?

Reply:
I've not pinned the pip version because ideally I shouldn't need to - if pip 25.* is Python 3.12 compatible then it's sensible we bump to it.

Python version should ideally be 3.12 in all repos - but for some others it's not possible to do that right now as there's other work that needs to happen (which I've raised followon tickets for)